src/includes/traits/Ac/ObUtils.php

```diff
@@ -348,14 +348,32 @@ public function outputBufferCallbackHandler($buffer, $phase)
-            $cache .= "\n".htmlspecialchars(sprintf(__('Expiration date: \t10s %2$s (based on your configured expiration time)', SLUG_TD), NAME, date('M jS, Y @ g:i a T', strtotime('+'.COMET_CACHE_MAX_AGE))));
-            $cache .= "\n".htmlspecialchars(sprintf(__('Time to generate this cache file: %4$s seconds', SLUG_TD), NAME, $this->is_404 ? '404 [error document]' : $this->salt_location, (IS_PRO && COMET_CACHE_WHEN_LOGGED_IN && $this->user_token ? '; '.sprintf(__('user token: %1$s', SLUG_TD), $this->user_token) : ''), $total_time, date('M jS, Y @ g:i a T')));
-            $cache .= "\n".htmlspecialchars(sprintf(__('Generated via: \t10s %6$s', SLUG_TD), NAME, $this->is_404 ? '404 [error document]' : $this->salt_location, (IS_PRO && COMET_CACHE_WHEN_LOGGED_IN && $this->user_token ? '; '.sprintf(__('user token: %1$s', SLUG_TD), $this->user_token) : ''), $total_time, date('M jS, Y @ g:i a T'), $via));
-            $cache .= "\n".' -->';
+
+            $cache .= $notes;
+
+            $notes = []; // Initialize array.
+            $notes['Cache File Information and Statistics'] = "\n".'++++++++++++++++'.htmlspecialchars(sprintf(__(' ', SLUG_TD), NAME, str_replace(WP_CONTENT_DIR, '', $this->is_404 ? $this->cache_file_404 : $this->cache_file)));
+            $notes['File Path'] = "\n".htmlspecialchars(sprintf(__('%2$s', SLUG_TD), NAME, str_replace(WP_CONTENT_DIR, '', $this->is_404 ? $this->cache_file_404 : $this->cache_file)));
+            $notes['Page URL']  = "\n".htmlspecialchars(sprintf(__('%2$s%3$s', SLUG_TD), NAME, $this->is_404 ? '404 [error document]' : $this->salt_location, (IS_PRO && COMET_CACHE_WHEN_LOGGED_IN && $this->user_token ? '; '.sprintf(__('user token: %1$s', SLUG_TD), $this->user_token) : ''), $total_time, date('M jS, Y @ g:i a T')));
+            $notes['Date generated'] = "\n".htmlspecialchars(sprintf(__('%5$s', SLUG_TD), NAME, $this->is_404 ? '404 [error document]' : $this->salt_location, (IS_PRO && COMET_CACHE_WHEN_LOGGED_IN && $this->user_token ? '; '.sprintf(__('user token: %1$s', SLUG_TD), $this->user_token) : ''), $total_time, date('M jS, Y @ g:i a T')));
+            $notes['Expiration date'] = "\n".htmlspecialchars(sprintf(__('%2$s (based on your configured expiration time)', SLUG_TD), NAME, date('M jS, Y @ g:i a T', strtotime('+'.COMET_CACHE_MAX_AGE))));
+            $notes['Time to generate this cache file:'] = "\n".htmlspecialchars(sprintf(__('%4$s seconds', SLUG_TD), NAME, $this->is_404 ? '404 [error document]' : $this->salt_location, (IS_PRO && COMET_CACHE_WHEN_LOGGED_IN && $this->user_token ? '; '.sprintf(__('user token: %1$s', SLUG_TD), $this->user_token) : ''), $total_time, date('M jS, Y @ g:i a T')));
```

Comment:
You can exclude the `:` from the heading in this line. We can add this when each item is formatted below.